moveit_planners/ompl/ompl_interface/CMakeLists.txt

```diff
@@ -49,14 +52,25 @@ if(CATKIN_ENABLE_TESTING)
   target_link_libraries(test_state_space ${MOVEIT_LIB_NAME} ${OMPL_LIBRARIES})
   set_target_properties(test_state_space PROPERTIES LINK_FLAGS "${OpenMP_CXX_FLAGS}")
 
+  catkin_add_gtest(test_constrained_planning_state_space test/test_constrained_planning_state_space.cpp)
+  target_link_libraries(test_constrained_planning_state_space ${MOVEIT_LIB_NAME} ${OMPL_LIBRARIES} ${catkin_LIBRARIES})
+  set_target_properties(test_constrained_planning_state_space PROPERTIES LINK_FLAGS "${OpenMP_CXX_FLAGS}")
+
+  catkin_add_gtest(test_constrained_state_validity_checker test/test_constrained_state_validity_checker.cpp)
+  target_link_libraries(test_constrained_state_validity_checker ${MOVEIT_LIB_NAME} ${OMPL_LIBRARIES} ${catkin_LIBRARIES})
+  set_target_properties(test_constrained_state_validity_checker PROPERTIES LINK_FLAGS "${OpenMP_CXX_FLAGS}")
+
+  catkin_add_gtest(test_ompl_constraints test/test_ompl_constraints.cpp)
+  target_link_libraries(test_ompl_constraints ${MOVEIT_LIB_NAME} ${OMPL_LIBRARIES} ${catkin_LIBRARIES})
+
+  # Unfortunately it is not possible to test the planning context manager without introducing a ROS dependency.
   find_package(rostest REQUIRED)
   find_package(eigen_conversions REQUIRED)
-
   add_rostest_gtest(test_planning_context_manager
     test/test_planning_context_manager.test
     test/test_planning_context_manager.cpp)
-  target_link_libraries(test_planning_context_manager ${MOVEIT_LIB_NAME} ${OMPL_LIBRARIES} ${catkin_LIBRARIES} ${eigen_conversions_LIBRARIES})
-
+  target_link_libraries(test_planning_context_manager ${MOVEIT_LIB_NAME} ${OMPL_LIBRARIES} ${catkin_LIBRARIES})
+  
```

Comment:
```suggestion

```

Reply:
This change must have sneaked in when solving the merge conflicts. I linked the library again, it is required for the test. Or did you mean something else with the suggestions?

Reply:
I just suggested deleting the trailing whitespace. Obviously, if `${eigen_conversions_LIBRARIES}` is still needed, then don't delete that :-).

Reply:
Ah, ok. I have a single whitespace before every "add test" command now.